packages/block-editor/src/components/inserter/block-types-tab.js

```diff
@@ -83,6 +102,19 @@ export function BlockTypesTab(
 	// Hide block preview on unmount.
 	useEffect( () => () => onHover( null ), [] );
 
+	const getRootBlockTitle = () => {
+		const blockName = getBlockName( rootClientId );
+		// Find the title within the list of all possible blocks.
+		const block = allItems.find( ( item ) => item.name === blockName );
+		return block
+			? sprintf(
+					// translators: %s: Block title, e.g: "List", "Buttons", "Group"
+					__( '%s Block' ),
```

Comment:
We don't need "Block" appended here. The block name itself will be enough context. "List", "Columns", "Buttons", etc.